CIP-XXXX/ts-api.md

```diff
@@ -0,0 +1,154 @@
+### Full API
```

Comment:
I think we may want to leave out any specific details related to the wallet API here as that seems more relevant to the wallet connector CIP. Perhaps you could just list out each method, omitting the  `api.query.` namespace?